src/tests/functional/plugin/conformance/subgraphs_dumper/src/ops_cache.cpp

```diff
@@ -159,6 +159,28 @@ float OPCache::get_size_of_cached_ops() {
     return size;
 }
 
+std::string getOpVersion(const ngraph::NodeTypeInfo& type_info) {
+    static std::vector<ngraph::OpSet> opsets;
+    if (opsets.size() == 0) {
+        opsets.push_back(ngraph::get_opset1());
+        opsets.push_back(ngraph::get_opset2());
+        opsets.push_back(ngraph::get_opset3());
+        opsets.push_back(ngraph::get_opset4());
+        opsets.push_back(ngraph::get_opset5());
+        opsets.push_back(ngraph::get_opset6());
+        opsets.push_back(ngraph::get_opset7());
+        opsets.push_back(ngraph::get_opset8());
+        opsets.push_back(ngraph::get_opset9());
+        opsets.push_back(ngraph::get_opset10());
+    }
```

Comment:
Please use this method:
https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/opsets/opset.hpp#L193